### ISSUE TITLE
XEK branches

### ISSUE THREAD

**Issue by Fearuncle** [sars-cov-2-variants/lineage-proposals#2179]
![image](https://github.com/user-attachments/assets/38536686-65e9-44ab-b1e0-2ed4e9ce52c9)
tree: https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_1502f_9795e0.json?f_userOrOld=uploaded%20sample
- [x] _Designated_ **XEK.1** and **XEK.1.1**  [XEK + ORF1a:R1170H(114 in 16 Countries) + 15 with S:S680P
Gisaid: !T21568A & G3774A & C593T]
- [x] _Designated_ **XEK.2** [XEK + ORF3a:A23S(46 in 8 countries)
Gisaid: C13381T, G25459T, C3505T]
- [ ] XEK + ORF1b:I2320T, ORF1a:T4249I(24 in 3 countries) + 4 with Orf8:G8* + S:A942S
Gisaid: T20426C, C593T, C897A
- [ ] XEK + S:M177I (5 in 2 countries) 
Gisaid: G16662A,G22093C,T3565C
- [ ] XEK + S:E1195Q & S:E156D (6 in 1 countries)
Gisaid: G25145C ,C11563T, C28657T

**Comment by Fearuncle** [comment]
![image](https://github.com/user-attachments/assets/77999ab8-938d-4520-823d-3cffee25cc34)

**Comment by FedeGueli** [comment]
there maybe a branch with S:A>D264H with two samples Germany/ Wales Query:   C9286T,22352C

**Comment by Fearuncle** [comment]
There are 5 with S:M177I, and 6 with S:E1195Q & S:E156D

**Comment by FedeGueli** [comment]
> There are 5 with S:M177I, and 6 with S:E1195Q & S:E156D

Yes! i am tracking the 156D one with G25145C  ,C11563T, C28657T,
Saw the other one but still not tracking it please add both to the proposal!

**Comment by FedeGueli** [comment]
Suggested query for M117I : G16662A,G22093C,T3565C

**Comment by FedeGueli** [comment]
@Fearuncle we have extended the. XEC spike diversity issue to XEK so please comment there, @xz-keg added there the M177I branch  check it out: https://github.com/sars-cov-2-variants/lineage-proposals/issues/2088

**Comment by FedeGueli** [comment]
XEK + ORF1a:R1170H(114 in 16 Countries) + 15 with S:S680P Designated XEK.1 and XEK.1.1 via https://github.com/cov-lineages/pango-designation/commit/4d37ed8d11f5e3fec356943bf2484b21a0b4b183
